src/cobra/util/solver.py

```diff
@@ -50,11 +50,11 @@
 solvers = {
     match.split("_interface")[0]: getattr(optlang, match)
     for match in dir(optlang)
-    if "_interface" in match
+    if "_interface" in match and match != "matrix_interface"
 }
 
 # Defines all the QP solvers implemented in optlang.
-qp_solvers = ["cplex", "gurobi", "osqp"]
+qp_solvers = ["cplex", "gurobi", "hybrid", "osqp"]
```

Comment:
Since the hybrid interface is a replacement for the direct OSQP interface in optlang, do you really want to keep it here in the list?

Reply:
Yeah it was more for deprecating it's use. But you are right, makes no sense here. I will fix it.